RecipeApp/Modules/RecipeBook/FlowCoordinator/RecipeBookCoordinator.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RecipeBookCoordinator.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import UIKit
+
+class RecipeBookCoordinator: Coordinator {
+    
+    var navigationController: UINavigationController
+    var favoriteService: FavoriteServiceProtocol
+
+    init(navigationController: UINavigationController, favoriteService: FavoriteServiceProtocol) {
+       self.navigationController = navigationController
+       self.favoriteService = favoriteService
+   }
+
+    func start() {
+        let recipeBookViewModel = RecipeBookViewModel(favoriteService: favoriteService)
+        let recipeBookViewController = RecipeBookViewController.create(viewModel: recipeBookViewModel)
+        recipeBookViewController.coordinator = self
```

Comment:
I don't believe this coordinator property is used anymore. If it is then it shouldn't be since you're using view models now!